awx/main/models/indirect_managed_node_audit.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.postgres.fields import ArrayField
```

Comment:
It doesn't look like this is being used. I know what it was intended for. Maybe we should still add that. I'm not sure.